src/controllers/article.controller.js

```diff
@@ -289,6 +289,7 @@ export default {
       if (error.status === 404) {
         return Helper.errorResponse(response, error.status, error.message);
       }
+      console.log(error);
```

Comment:
Great work Sam. You forgot to take out the console.log statement

Reply:
Thanks for pointing that out. I will take it out and would double-check if there are some more.